Architecture.md

```diff
@@ -71,6 +71,79 @@
 
 ***TODO: [GIT] how are the projects stored and how is authentication handled upon push***
 
+As all PaaS providers, PaaSTech needs to store the code of the projects created by its clients. This is done using a Git repository, which is created upon project creation. The repository is then exposed to the client, who can push its code to it.
+
+We chose to use Git to store the code of the projects, because it is the most used version control system, and is the most adapted to store code. Using it makes the integration seamless for users who are probably already using git.
+
+##### Creating and deleting repositories
+
+When a user initializes a new project, a new repository has to be created, it also has to be deleted when the project is deleted.
+
+The repository is created using the [git init](https://git-scm.com/docs/git-init) command. The repository is then stored in a directory, which is named after the **project's id**. It is stored on the host machine.
```

Comment:
```suggestion
The repository is created using the [`git init`](https://git-scm.com/docs/git-init) command. The repository is then stored in a directory, which is named after the **Project's id**. It is stored on the host machine.
```

Reply:
fixed in ff0de43117079e021117d93e73349ac7ba12eee1